zephyr/module.yml

```diff
@@ -2,3 +2,11 @@ name: hal_st
 build:
   cmake: .
   kconfig-ext: True
+blobs:
+  - path: img/bluetooth/steval-mkboxpro/dtm.bin
+    sha256: f65b59363482a1122d4155ceb843b1fe7bb5612496f81feb50746f110fc27ebf
+    type: img
+    version: '1.0.0'
+    license-path: zephyr/blobs/license.md
+    url: https://github.com/STMicroelectronics/stsw-mkbox-bleco/raw/v1.0.0/img/steval-mkboxpro/DTM_SPI_WITH_UPDATER_CONTROLLER.bin
```

Comment:
If this fw is specific to this board, it should be put in hal_stm32.

Reply:
Yes, it is